### ISSUE TITLE
Select + MMB copy&paste only the first character

### ISSUE THREAD

**Issue by maek** [glowing-bear/glowing-bear#921]
When I select some text of a message and then I try to paste it in any other application using middle mouse button it copies only the first character of the text.

Chromium 57.0.2987.110 (Build) (a 64 bit), Arch Linux.

**Comment by lorenzhs** [comment]
Yeah Glowing Bear's copy&paste behaviour is terrible. Someone with more knowledge of the DOM and browsers needs to look at this, though, I have no idea how to fix it.
